index.js

```diff
@@ -17,3 +17,112 @@ close.addEventListener('click', () => {
+  technologies: ['HTML', 'CSS', 'JavaScript','Ruby'],
+  popBtn1: 'see live',
+  popBtn2: 'see scource'
+
+}
+];
+const p1 = document.querySelector('#p1');
+function setpopup(btnID)
+{
+ li='';
+  project.filter(project => project.id == btnID).forEach((e)=>
+  {
+const popup_window = document.getElementById('popup_window');
+  e.technologies.forEach((project) =>
+  {
+    li+= `<li class="lang" id="lang1">${project}</li>`
+  });
+const popupCode = `
+<div class="worksec1 worksec1d1" id="worksec1d">
+    <div class="worksf">
+    <div id="popupHead">
+    <button onclick="closeWinfun()" id="closeWindow"><img id="close-img" src="./assets/images/Enabled.svg" alt="Close Cover"></button>
+    <h2 class="desktop-hidden" id="title">${e.name}</h2>
+    <br> </div>
+    <div>
+     <ul class="connect-via" id="dev">
+         <li class="canopy">Canopy </li>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp;&nbsp; Back End Dev</li>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp; &nbsp; 2015</li>
+     </ul>
+    </div>
+     <div class="worksf dsk-width1" >
+        <img src=${e.featuredImage} id="c1" alt="pic">
+        </div>
+        <div class="pop-dsk-container">
+     <div class="discription">
+       <p class="discpara" id="discparapop">${e.discription}</p>
+     </div>
+     <div id="pop-group-tech-live">
+     <div class="social-mv dsk-pop" id="technology">
+     <ul class="connect-via desktop-mv-lang ">
+        ${li}
+     </ul> </div>
+     <div id="popup-btn">
+     <button type="button" class="project1-btn btn-hover" id="btn-project">${e.popBtn1} <img src="./assets/images/btniconexp.svg" alt="exp"></button>
+     <button type="button" class="project1-btn btn-hover id="btn-project">${e.popBtn2}<img src="./assets/images/btnicongithub.svg" alt="exp"></button>
+    </div>
+    </div>
+    </div>
+    <div>
+    </div>
+    `;
+    popup_window.innerHTML = popupCode;
+     
+  document.getElementById('title').style.cssText = "display: block;";
+  document.getElementById('technology').style.cssText ="padding-top: 430px";
+  document.getElementById('closeWindow').style.cssText ="float: right; padding-bottom: 10px width: 24px; height: 24px; background-color: white; border: 0px";
+  document.getElementById('cover').style.cssText = 'display: block; z-index: 3; width: 100%; height: 100%; background: #C1C7D0;';
+  document.getElementById('menu').style.cssText = 'display: none'; 
+  document.getElementById('body').style.cssText = 'overflow: hidden;';
+  document.getElementById('btn-project').style.cssText = 'justify-content: space-between;';
+ document.getElementById("c1").style.cssText="display: block; margin: 8px; margin-top: -45px; width: 95%;";
+  }
+  )
+};
```

Comment:
- [ ] Kindly try to match the Figma design for the list of technologies in the modal by decreasing the spacing between each technology and increasing padding of each technology.

| Your design | Figma design |
| --- | --- | 
| ![image](https://user-images.githubusercontent.com/24703171/156570833-9e3509b9-3c00-463d-9876-7b9aacdc2ec6.png) | ![image](https://user-images.githubusercontent.com/24703171/156570904-b236b1e2-c40f-4df1-b547-50d48cea7004.png) |

- [ ] Kindly remove the red border that surrounds your modal to match the Figma design.
![image](https://user-images.githubusercontent.com/24703171/156571733-90eb8659-151b-45ef-89a9-8bd3ae59c0ed.png)

- [ ] Kindly increase the left and right spacing between the modal and the `see live` and `see source` buttons in the mobile version and make the `see source` button in a single line to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/156572090-292aa602-c2d7-42f8-811b-99b7d3033f4c.png) | ![image](https://user-images.githubusercontent.com/24703171/156572153-bc232a23-bc43-4357-a1e5-ac08cd9385f1.png) |